spyder/plugins/application/widgets/status.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Kite completions.
+"""
+
+# Standard library imports
+import logging
+
+# Third party imports
+from qtpy.QtCore import Slot
+
+# Local imports
+from spyder.api.widgets.status import StatusBarWidget
+from spyder.config.base import _
+from spyder.plugins.application.widgets.install import (
+    UpdateInstallerDialog)
+from spyder.utils.icon_manager import ima
+
+logger = logging.getLogger(__name__)
+
+
+class ApplicationUpdateStatus(StatusBarWidget):
+    """Status bar widget for Application update status."""
+    BASE_TOOLTIP = _("Application update status")
+    DEFAULT_STATUS = _('not reachable')
```

Comment:
Maybe we can change this default status to something like the current Spyder version installed.